src/fr/send.md

```diff
@@ -1 +1,291 @@
+
+Si la demande a été refusée, le corps de la réponse est “json”, consultez le tableau ci-dessous pour plus de détails.
+
+|status_code|message|Comment réparer|
+|:---|:---|:---|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Impossible d’envoyer à ce destinataire à l’aide d’une clé API d’équipe uniquement"`<br>`}]`|Utiliser le bon type de [clé API](keys.md)|
+|`400`|`[{`<br>`"erreur": "BadRequestError",`<br>`"message": "Impossible d’envoyer cette notification à ce destinataire lorsque le service est en mode d’essai`<br>`}]`|Votre service ne peut pas envoyer cette notification en mode d’essai|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message" : "Type de fichier non pris en charge '(TYPE DE FICHIER)'. Les types pris en charge sont les suivants : '(TYPES AUTORISÉS)"`<br>`}]`|Mauvais type de fichier. Vous ne pouvez télécharger que des fichiers .pdf, .csv, .txt, .jpeg, .png, .doc, .docx, .xls, .xlsx, .rtf ou .odt
+|`400`|`[{`<br>`"erreur": "BadRequestError",`<br>`"message": "Le fichier n’a pas réussi l’analyse antivirus"`<br>`}]`|Le fichier contient un virus|
+|`400`|`[{`<br>`"erreur": "BadRequestError",`<br>`"message": "L’envoi de fichiers par courriel n’a pas été configuré – ajoutez les coordonnées de votre service"`<br>`}]`|Voir comment ajouter les coordonnées à la page de téléchargement de fichier|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Error : Votre horloge système doit être précise dans les 30 secondes"`<br>`}]`|Vérifiez votre horloge système|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Jeton non valide : Clé API introuvable"`<br>`}]`|Utilisez la bonne clé API. Consultez les [clés API](keys.md) pour plus de renseignements|
+|`429`|`[{`<br>`"erreur": "RateLimitError",`<br>`"message": "Dépassement du débit maximal pour le type de clé TEAM/TEST/LIVE de 3000 requêtes par 60 secondes"`<br>`}]`|Reportez-vous à [Débits maximaux API](limits.md) pour plus de renseignements|
+|`429`|`[{`<br>`"erreur": "TooManyRequestError",`<br>`"message": "Dépassement des limites d’envoi (NOMBRE MAXIMAL) pour aujourd’hui"`<br>`}]`|Reportez-vous à [limites du service](limits.md) pour le nombre maximal|
+|`500`|`[{`<br>`"erreur": "Exception",`<br>`"message": "Erreur interne du serveur"`<br>`}]`|GC Notification n’a pas pu traiter la demande, renvoyez votre notification.|
+
+## Envoyer un message texte
+
+```
+POST /v2/notifications/sms
+```
+
+### Corps de la demande
+
+```json
+{
+  "phone_number": "+447900900123",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a"
+}
+ ```
+
+### Arguments
+
+**phone_number (obligatoire)**
+
+Numéro de téléphone du destinataire du message texte.
+
+**template_id (obligatoire)**
+
+Pour rechercher l’ID de gabarit :
+
+1. [Connectez-vous à GC Notification](https://notification.canada.ca/sign-in?lang=fr).
+1. Accédez à la page __Gabarits__ et sélectionnez le gabarit approprié.
+1. Sélectionnez __Copier template ID dans le presse-pappier__.
+
+**personnalisation (facultatif)**
```

Comment:
This is the second time we have sections like this. It's weird because we have words in French that don't watch the technical side personalisation vs `personalisation` but at the same time we have `template_id` and `sms_sender_id` even in French.

I think we need to be consistent and:
- put section titles in plain words in French and keep keys in English in JSON
- put sections titles in English to match the API but maybe quote (`sms_sender_id` in the section title)

Reply:
This is great! Will apply! :D